PA_SPA_1-Timing_Analysis_with_Power_for_Password_Bypass.ipynb

```diff
@@ -400,11 +400,11 @@
     "def checkpass(trace, i):\n",
     "    if PLATFORM == \"CWNANO\":\n",
     "        #There's a bit of jitter\n",
-    "        return (trace[228 + 11*i] < 3 and trace[227 + 11*i] < 0.3)\n",
+    "        return (trace[228 + 11*i] > 3 and trace[227 + 11*i] > 0.3)\n",
```

Comment:
Another thing that I spotted, is both "3" and "0.3" correct here? I don't own a CWNANO, so I cannot really check. But by just thinking about it it looks like both should be either "3" or that both should be "0.3"?